lib/repl.js

```diff
@@ -374,6 +374,17 @@ function REPLServer(prompt,
     };
   }
 
+  function prettifyStackTrace(err) {
+    if (err.stack) {
+      const regex = new RegExp(/\s+repl:/);
```

Comment:
I believe `new RegExp` is unnecessary in recent ES versions.

Reply:
I had originally written this inline below as 

```js
const idx = stack.findIndex((frame) => /\s+repl:/.test(frame));
```

Which worked as expected. It just wreaked havoc on my syntax highlighting. 😜  I guess that's not the best reason to use `new`. But it's all I've got.